src/pages/transactions/[id].js

```diff
@@ -41,6 +44,13 @@ export default function TransactionDetails() {
         <h2>Description</h2>
         <div>{transaction.description}</div>
       </div>
+      {mode === "delete" && (
+        <Modal
+          message="Really?.. Delete?"
```

Comment:
This message does not meet your acceptance criteria. It should inform the user that the deletion is permanent and cannot be undone.

Reply:
You're correct! This was for testing. My mistake.

Reply:
change wording!